src/schema.ts

```diff
@@ -110,32 +110,32 @@ const typeDefs = `#graphql
     """
     Updates an address by id
     """
-    updateAddress(_id: ID!, zipCode: Int, street: String, houseNumber: Int): Address
+    updateAddress(_id: ID!, zipCode: Int, street: String, houseNumber: Int, token: String!): Address
     """
     Deletes an address by id
     """
-    deleteAddress(_id: ID!): Address
+    deleteAddress(_id: ID!, token: String!): Address
     """
     Create Admin for company
     """
-    createAdmin(firstName: String!, lastName: String!, email: String!, password: String!, role: Role!, zipCode: Int!, street: String!, houseNumber: Int!, companyId: ID!): User
+    createAdmin(firstName: String!, lastName: String!, email: String!, password: String!, role: Role!, zipCode: Int!, street: String!, houseNumber: Int!, companyId: ID!, token: String!): User
     
     """
     Creates a new company
     """
-    createCompany(name: String!, description: String!, zipCode: Int!, street: String!, houseNumber: Int!, companyOwnerId: ID!): Company
+    createCompany(name: String!, description: String!, zipCode: Int!, street: String!, houseNumber: Int!, companyOwnerId: ID!, token: String!): Company
     """
     Updates a company by id
     """
-    updateCompany(_id: ID!, name: String, description: String, zipCode: Int, streetName: String, houseNumber: Int): Company
+    updateCompany(_id: ID!, name: String, description: String, zipCode: Int, streetName: String, houseNumber: Int, token: String!): Company
     """
     Deletes a company by id
     """
-    deleteCompany(_id: ID!): Company
+    deleteCompany(_id: ID!, token: String!): Company
     """
     Remove admin from company
     """
-    deleteAdmin(userId: ID!, companyId: ID!): Company
+    deleteAdmin(userId: ID!, companyId: ID!, token: String!): Company
```

Comment:
I assume this is the company admin we are deleting here. Could we rename it to be clearer?